### PR TITLE
Update samples for packaging

### PR DESCRIPTION
**Pull request by bbowman**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc-samples/19)

### CODE REVIEW

File: Apps/BitcoinWallet/BitcoinWallet.vsimporter/BitcoinWallet-WinStore10/Package.appxmanifest [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  <Identity
+    Name="e3660482-e8b1-4343-addc-44d336ad21db"
+    Publisher="CN=islandwood"
+    Version="1.0.0.0" />
+
+  <mp:PhoneIdentity PhoneProductId="e3660482-e8b1-4343-addc-44d336ad21db" PhonePublisherId="00000000-0000-0000-0000-000000000000"/>
+
+  <Properties>
+    <DisplayName>BitcoinWallet</DisplayName>
+    <PublisherDisplayName>bbowman</PublisherDisplayName>
```
Comment (memontic-ms) [inline Apps/BitcoinWallet/BitcoinWallet.vsimporter/BitcoinWallet-WinStore10/Package.appxmanifest]: Might want to change all these unless you want to be famous.

**Review by memontic-ms — APPROVED** [review]

File: Scenarios/FileActivation/FileActivationSample/FileActivation.vsimporter/FileActivation-WinStore10/Generated Files/App.g.h [hunk 2]
```diff
@@ -1,27 +0,0 @@
-﻿#pragma once
-//------------------------------------------------------------------------------
-//     This code was generated by a tool.
-//
-//     Changes to this file may cause incorrect behavior and will be lost if
-//     the code is regenerated.
-//------------------------------------------------------------------------------
-
-#include "XamlTypeInfo.g.h"
```
Comment (jaredhms) [inline Scenarios/FileActivation/FileActivationSample/FileActivation.vsimporter/FileActivation-WinStore10/Generated Files/App.g.h:9]: >XamlTypeInfo [](start = 10, length = 12)

These files didn't exist for any of the other apps?

**Comment by jaredhms** [comment]
:shipit:

File: Scenarios/FileActivation/FileActivationSample/FileActivation.vsimporter/FileActivation-WinStore10/Generated Files/App.g.h (hunk 2 above)
Comment (bbowman) [reply Scenarios/FileActivation/FileActivationSample/FileActivation.vsimporter/FileActivation-WinStore10/Generated Files/App.g.h:9]: nope!

---
In reply to: [107052986](https://github.com/Microsoft/WinObjC-Samples/pull/19#discussion_r107052986) [](ancestors = 107052986)

File: Apps/BitcoinWallet/BitcoinWallet.vsimporter/BitcoinWallet-WinStore10/Package.appxmanifest (hunk 1 above)
Comment (bbowman) [reply Apps/BitcoinWallet/BitcoinWallet.vsimporter/BitcoinWallet-WinStore10/Package.appxmanifest]: this is my least favorite thing ever.

---
In reply to: [107052264](https://github.com/Microsoft/WinObjC-Samples/pull/19#discussion_r107052264) [](ancestors = 107052264)

**Pull request merged**